src/filebrowser/file-browser-dialog.cpp

```diff
@@ -68,6 +68,15 @@ bool inline findConflict(const QString &name, const QList<SeafDirent> &dirents)
     return found_conflict;
 }
 
+QString appendTrailingSlash(const QString& input) {
+    if (!input.endsWith('/')) {
+        QString ret = input;
+        return ret.append('/');
```

Comment:
Why do we need this `ret` variable?

The function can be a one-liner like  `input.endsWith("/") ? input : input + "/"`